subworkflows/local/fusion.nf

```diff
@@ -36,7 +36,7 @@ workflow FUSION {
     ch_versions = Channel.empty()
     //fasta = params.fasta
     //gene_bed = params.metafusion_gene_bed
-    //gene_info = params.metafusion_gene_info
+    clinicalgenes = params.clinicalgenes
```

Comment:
you should pass in params.clinicalgenes in the `take` block like all the other resource params.